packages/mui-material/src/Slider/Slider.d.ts

```diff
@@ -55,13 +56,14 @@ export type SliderTypeMap<
   defaultComponent: D;
 }>;
 
+export { SliderValueLabelProps } from '@mui/base/SliderUnstyled';
+
 type SliderRootProps = NonNullable<SliderTypeMap['props']['componentsProps']>['root'];
 type SliderMarkProps = NonNullable<SliderTypeMap['props']['componentsProps']>['mark'];
 type SliderMarkLabelProps = NonNullable<SliderTypeMap['props']['componentsProps']>['markLabel'];
 type SliderRailProps = NonNullable<SliderTypeMap['props']['componentsProps']>['rail'];
 type SliderTrackProps = NonNullable<SliderTypeMap['props']['componentsProps']>['track'];
 type SliderThumbProps = NonNullable<SliderTypeMap['props']['componentsProps']>['thumb'];
-type SliderValueLabelProps = NonNullable<SliderTypeMap['props']['componentsProps']>['valueLabel'];
```

Comment:
I think that this was wrong. The props in here end up being none required while they should be.

Reply:
cc @michaldudak

Reply:
:+1: Agree. Components in Base don't use the same type for these (unless I missed a case).
In general nothing should be required in `componentsProps.*`, but there are many non-optional props in types of props supplied to `components.*`.